devicemodel/include/vga.h

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (C) 2022 Intel Corporation. All rights reserved.
```

Comment:
Remove the `All rights reserved` string

Reply:
the All rights reserved string is not a must, but it does no harm. So my recommendation is: 
* if it's already there, don't bother to remove it. 
* If it's a new file, don't add it. 
In some cases, we need to keep it though. for example, we copied a snippet from https://github.com/oasis-tcs/virtio-spec/blob/master/virtio-gpu.tex, which requires we keep its copyright line including "all rights reserved".

Reply:
I don't see this `BSD-2-Clause-FreeBSD` license in the link you provided above. Where are those snippets coming from?

Reply:
Hi @yaomichael ,

I sought guidance from the legal team and here is their feedback.

> the All rights reserved string is not a must, but it does no harm. So my recommendation is:
> 
>     * if it's already there, don't bother to remove it.

They recommend we actively remove it (from the Intel copyright line only of course).

> 
>     * If it's a new file, don't add it.

Yes

>       In some cases, we need to keep it though. for example, we copied a snippet from https://github.com/oasis-tcs/virtio-spec/blob/master/virtio-gpu.tex, which requires we keep its copyright line including "all rights reserved".

True for all other, non-Intel copyright lines.